os-virt.te

```diff
@@ -20,6 +21,11 @@ allow virtlogd_t systemd_logind_t:dbus send_msg;
 allow systemd_logind_t virtlogd_t:dbus send_msg;
 allow virtlogd_t systemd_logind_inhibit_var_run_t:fifo_file write;
 
+#============= virtlogd_t ==============
```

Comment:
(nit, non-blocking) We also have virtlogd_t rules elsewhere (e.g. right above) so I would remove the header. The comment below with the link to the JIRA is super helpful though, thanks!